src/components/Map/LeafletMap.tsx

```diff
@@ -0,0 +1,55 @@
+import turfBbox from "@turf/bbox";
+import L from "leaflet";
+import "leaflet/dist/leaflet.css";
+import React from "react";
+import { Map } from "react-leaflet";
+import MapBoxGLLayer from "~/components/Map/MapBoxGLLayer";
+import MarkerCluster from "~/components/Map/MarkerCluster";
+import SearchControl from "~/components/Map/SearchControl";
+import API from "~/lib/api";
+import axiosInstance from "~/lib/axiosInstance";
+
+delete L.Icon.Default.prototype._getIconUrl;
+
+L.Icon.Default.mergeOptions({
+  iconRetinaUrl: require("leaflet/dist/images/marker-icon-2x.png"),
+  iconUrl: require("leaflet/dist/images/marker-icon.png"),
+  shadowUrl: require("leaflet/dist/images/marker-shadow.png")
+});
+
+class LeafletMap extends React.Component<{}, { points: any; bbox: any }> {
```

Comment:
Würde empfehlen die types direkt unter den import statements festzulegen. Dann sieht man direkt eingangs, welchen State bzw. Props die Component benötigt/nutzt.

Sowas wie
```
type Props = {};

type State = {
  points: any;
  bbox: any;
};
```

und in Z20 dann:
`class LeafletMap extends React.Component<Props, State>`

Bekommen wir für **points** und **bbox** keine genauen types?

Reply:
Ah, wahrscheinlich doch, sollte möglich sein!